src/Lykke.HftApi.Services/AssetsHttpClient.cs

```diff
@@ -56,6 +141,17 @@ public async Task<IReadOnlyList<AssetPair>> GetAllAssetPairsAsync()
                 MinOppositeVolume = x.MinInvertedVolume
             }).ToArray();
         }
+
+        public async Task<HashSet<string>> GetAssetsAvailableForClientAsync(string clientId)
+        {
+            var request = new HttpRequestMessage(HttpMethod.Get, $"/api/v2/clients/{clientId}/asset-ids?isIosDevice=true");
```

Comment:
why `isIosDevice`?

Reply:
This is how it's done in API v2: https://github.com/LykkeCity/LykkeWalletAPI/blob/master/LkeServices/AssetsHelper.cs#L103